src/poll.warden.cpp

```diff
@@ -57,13 +57,13 @@ void felspar::io::poll_warden::run_until(felspar::coro::coroutine_handle<> coro)
         } else if (pr > 0) {
             continuations.clear();
             for (auto events : iops) {
-                if (events.revents & POLLIN) {
+                if (events.revents & (POLLIN | POLLERR | POLLNVAL | POLLPRI)) {
```

Comment:
I don't think so, but I've never needed this sort of data. If somebody who does know what they're doing does a POLLIN (presumably they'd be reading rather than writing?) then I guess they'd want to wake up. Dunno